worldengine/cli/main.py

```diff
@@ -395,9 +394,12 @@ def main():
 
     if args.seed is not None:
         seed = int(args.seed)
+        if not 0 <= seed <= 65535:  # platec might not be able to handle integers larger than uint16; TODO: check this
+            print("Seed has to be in the range of 0 and 65535.")
```

Comment:
platec_api_create here is getting a long:

https://github.com/Mindwerks/plate-tectonics/blob/master/src/platecapi.hpp#L26

I think a long could have any possible size depending on the compiler, the platform etc.

Then in the python bindings we actually store the seed in an unsigned int...

https://github.com/Mindwerks/plate-tectonics/blob/master/pybindings/platec_src/platecmodule.cpp#L19

So I would suggest we stay safe and limit the seed to 65535. Or we revise the whole thing to use platform independent types, which I am not super sure we can use in the python bindings anyway.